### PR TITLE
Update Boost documentation

### PR DESCRIPTION
**Pull request by AlenaSviridenko**
# Description
Added a special note for setting BOOST_ROOT variable since CMake requires it.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Validate-Boost.ps1 [hunk 1]
```diff
@@ -55,6 +55,10 @@ foreach ($BoostTool in $BoostTools)
 $CMakeFindBoostInfo = @"
 
 #### _Notes:_
+
+> ``BOOST_PATH`` was deprecated on images but it is required by CMake. Please make sure you set this variable
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-Boost.ps1:59]: I don't think that we should write `was deprecated` in documentation. it is more for changelog :)
I would consider something like `BOOST_ROOT` environment variable is not set up on the images.

Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-Boost.ps1:59]: `BOOST_PATH` -> `BOOST_ROOT`

Comment (AlenaSviridenko) [reply images/win/scripts/Installers/Validate-Boost.ps1]: thanks, updated

Comment (AlenaSviridenko) [reply images/win/scripts/Installers/Validate-Boost.ps1]: good catch, thank you

**Comment by miketimofeev** [comment]
Do we need to provide an example how to add this env variable? Or link to the docs at least

**Comment by AlenaSviridenko** [comment]
> Do we need to provide an example how to add this env variable? Or link to the docs at least

Actually, I think this is more of a warning not to forget to set it, and how to set an environment variable on Windows is a bit out of scope of this note.

**Pull request merged**